pkg/sql/colexec/colexecagg/COLEXEC.bzl

```diff
@@ -0,0 +1,70 @@
+# Map between target name and relevant template.
```

Comment:
I'd prefer to just put all of this in `pkg/sql/colexec/colexecagg/BUILD.bazel`. `eg_go_filegroup` and `gen_eg_go_rules` are not reusable, so defining them as utility functions doesn't provide any value. Just makes things more complicated for no reason.

Reply:
I am applying the same same treatment as 52c5f51. By pulling that inside `pkg/sql/colexec/colexecagg/BUILD.bazel` you mean follow the same logic that is already there in this PR or creating one `gen_rule` per file?